app/models/user_event.rb

```diff
@@ -0,0 +1,4 @@
+class UserEvent < ActiveRecord::Base
+	belongs_to :users
```

Comment:
user and event should be singular because 1 UserEvent can be connected with only 1 User and Event.